app/subscriber/src/features/home/Home.tsx

```diff
@@ -63,36 +51,25 @@ export const Home: React.FC = () => {
     return value;
   }, [appSettings]);
 
-  const homePage = React.useMemo(() => {
-    if (!featuredItemId) return;
-    return getBooleanActionValue(featuredItemId);
-  }, [featuredItemId]);
-
-  React.useEffect(() => {
-    if (!!homePage && !!filter.startDate && !isReady && !!featuredItemId) {
-      setIsReady(true);
-    }
-  }, [homePage, filter.startDate, isReady, featuredItemId]);
-
   React.useEffect(() => {
     // stops invalid requests before filter is synced with date
-    if (!isReady) return;
-    fetchResults(
-      generateQuery(
-        filterFormat({
-          ...settings,
-          actions: !!homePage ? [homePage] : [],
-          contentTypes: !!contentType ? filter.contentTypes : [],
-          startDate: filter.startDate,
-          endDate: filter.endDate,
-          mediaTypeIds: filter.mediaTypeIds ?? [],
-          sourceIds: filter.sourceIds ?? [],
-        }),
-      ),
-    );
-    // only run when filter is ready, and when filter.startDate changes
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [isReady, filter.startDate, featuredItemId]);
+    if (!!featuredItemId && !!filter.startDate) {
```

Comment:
Why complicate things and create a separate state/effect.  Just check if this effect should be actioned.